File: adventofcode-2016/src/test/java/nl/mvdr/adventofcode/adventofcode2016/day14/Part2Test.java
package nl.mvdr.adventofcode.adventofcode2016.day14;

import java.util.stream.Stream;

import org.junit.jupiter.params.provider.Arguments;

import nl.mvdr.adventofcode.solver.SolverTest;

class Part2Test extends SolverTest<Part2> {
    
    public Part2Test() {
        super(Part2.class);
    }
    
    static Stream<Arguments> testSolution() {
        return Stream.of(Arguments.of("22859", "example-day14.txt"));
    }
}

File: adventofcode-2025/src/test/kotlin/nl/mvdr/adventofcode/adventofcode2025/day02/Part2Test.kt
package nl.mvdr.adventofcode.adventofcode2025.day02

import nl.mvdr.adventofcode.solver.FunctionSolverTest
import org.junit.jupiter.params.provider.Arguments

class Part2Test: FunctionSolverTest<Long>(::solvePart2) {
    companion object {
        @JvmStatic
        fun testSolution(): List<Arguments> {
            return listOf(
                Arguments.of("4174379265", "example-day02.txt")
            )
        }
    }
}